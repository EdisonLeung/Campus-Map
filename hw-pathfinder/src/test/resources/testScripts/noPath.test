# Tests pathfinding on node with no path to another node

# Create Graph
CreateGraph noPath

# Add nodes
AddNode noPath a
AddNode noPath b
AddNode noPath c

# Add edges
AddEdge noPath b a 5.0
AddEdge noPath b c 5.0

# test pathfinding
FindPath noPath b a

FindPath noPath b c

FindPath noPath a b

FindPath noPath a c