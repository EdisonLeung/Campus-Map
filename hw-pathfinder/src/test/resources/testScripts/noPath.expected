# Tests pathfinding on node with no path to another node

# Create Graph
created graph noPath

# Add nodes
added node a to noPath
added node b to noPath
added node c to noPath

# Add edges
added edge 5.000 from b to a in noPath
added edge 5.000 from b to c in noPath

# test pathfinding
path from b to a:
b to a with weight 5.000
total cost: 5.000

path from b to c:
b to c with weight 5.000
total cost: 5.000

path from a to b:
no path found

path from a to c:
no path found